pkg/apis/planetscale/v2/vitessbackupstorage_types.go

```diff
@@ -103,6 +103,9 @@ type S3BackupLocation struct {
 	// Bucket is the name of the S3 bucket to use.
 	// +kubebuilder:validation:MinLength=1
 	Bucket string `json:"bucket"`
+	// Endpoint is the url endpoint for the S3 bucket.
+	// Default: User the endpoint associated with `Region` by the driver.
```

Comment:
We should refer to the JSON field name since these comments go into user-facing docs for the JSON API.

```suggestion
	// Default: Use the endpoint associated with `region` by the driver.
```